jsonHttpHandler/main.go

```diff
@@ -79,7 +96,22 @@ func (h JsonHttpHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
         ctx := r.Context()
         r = r.WithContext(context.WithValue(ctx, PathParamsKey, *pathParams))
 
-        routeData.Handler(h.globals)(w, r)
+        handler := routeData.Handler(h.globals)
+        middlewares := make([]Middleware, len(routeData.Middlewares))
+
+        copy(middlewares, routeData.Middlewares)
```

Comment:
This reverses the middlewares slice making sure that the first given middleware is executed as first and not last.

Reply:
Should we maybe add a comment? It seems as `copy` is not really descriptive in here

Reply:
Haha, you mean code below 🤦‍♂ . Cool in my side